### PR TITLE
[WIP] Sql connector

### PR DESCRIPTION
**Pull request by FlorEtcheverry**

### CODE REVIEW

File: src/flood_forecaster/data_model/river_station.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    id = Column(Integer, primary_key=True)
+    location_name = Column(String(100))
+    date = Column(DateTime)
+    level_m = Column(Integer)
+    station_number = Column(String(50))
+
+
+
+class PredictedRiverStation(Base):
```
Comment (maximepeter) [inline src/flood_forecaster/data_model/river_station.py:17]: Even if these class looks similar they have different name, why not going with `PredictedRiverLevel` for this class ?

File: src/flood_forecaster/data_model/river_station.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, String, DateTime
```
Comment (maximepeter) [inline src/flood_forecaster/data_model/river_station.py:1]: IMHO `river_station.py` is not descriptive enough. I mean this file contains class to represent the tables so maybe a name like `river_level_tables.py`.

File: src/flood_forecaster/utils/sql_connector.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import os
+
+from sqlalchemy import create_engine
+from sqlalchemy.engine import URL
+
+POSTGRES_PASSWORD = os.environ["POSTGRES_PASSWORD"]
```
Comment (maximepeter) [inline src/flood_forecaster/utils/sql_connector.py:6]: Are we sure this will be an environment variable ?

Comment (maximepeter) [reply src/flood_forecaster/utils/sql_connector.py:6]: Fair enough then, resolving :)

File: README.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ bash install/install.sh
 ```bash
 cd $BASE_PATH/src/data-extractor
 # Edit the file .env and add the following environment variables
-OPENAI_API_KEY=
+OPENAI_API_KEY=  # TODO remove
```
[the remark this answers is not in the document]
Comment (maximepeter) [reply README.md:40]: OK

File: src/flood_forecaster/data_model/river_station.py (hunk 2 above)
Comment (FlorEtcheverry) [reply src/flood_forecaster/data_model/river_station.py:1]: When working with ORMs in a project, the data model or using the [Repository pattern](https://dddinpython.hashnode.dev/implementing-the-repository-pattern-using-sqlalchemy) you abstract the database (and the fact that there are tables) from the business domain. Sadly there's no much time to do it as proper as I'd like to but that's the main reason for the naming.

File: src/flood_forecaster/data_model/river_station.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    date = Column(DateTime)
+    level_m = Column(Integer)
+    station_number = Column(String(50))
+
+
+
+class PredictedRiverStation(Base):
+    __tablename__ = 'predicted_river_level'
+
+    id = Column(Integer, primary_key=True)
```
Comment (ThierrySpetebroot) [inline src/flood_forecaster/data_model/river_station.py:20]: I would add:
 - model name for traceability
 - forecast_days to indicate the type of forecast it refers to (tomorrow's prediction, next week's, etc.)

**Review by maximepeter — APPROVED** [review]

**Pull request merged**